components/icons/images/tier1/arrow-expand-small.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" mirror-in-rtl="true">
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 1 18 18" mirror-in-rtl="true">
```

Comment:
@geurts May want to re-export this with the whole path shifted properly rather than modify the viewbox

Reply:
Nice catch — here's what I get when I fix the alignment and put it through svgomg: 

```
<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M6 12.96v-8c0-.4.2-.8.6-.9.4-.1.8-.1 1.1.2l4 4c.4.4.4 1 0 1.4l-4 4c-.3.3-.7.4-1.1.2-.4-.1-.6-.5-.6-.9Zm2-5.6v3.2l1.6-1.6L8 7.36Z" fill="#494C4E"/>
</svg>
```